pep-9999.rst

```diff
@@ -85,23 +85,71 @@ PyPI disables new distribution uploads for ``bdist_egg`` distributions.
 Removal Process
 ===============
 
-This PEP proposes a removal process based on :pep:`527`, but with the following
-modifications:
+This PEP does **NOT** propose removing any existing files from PyPI, only
+disallowing new ones from being uploaded.
 
-* During the deprecation period, *all* packages will
-  `continue to be allowed <https://www.youtube.com/watch?v=XNyUALnj8V0>`_
-  to upload ``bdist_egg`` distributions, not just those that have previously
-  uploaded ``bdist_egg`` distributions.
-* During the deprecation period, uploading a ``bdist_egg`` distribution to PyPI
-  will trigger an email to all maintainers of the project, rather
-  than sending a single email at the beginning of the deprecation period.
+PyPI will provide a deprecation period of one month. At the beginning
+of the deprecation period, maintainers of projects that already have egg
+distributions will receive a one-time email informing them of the upcoming
+restrictions on egg distribution uploads.
 
-After the deprecation period, PyPI will refuse to accept a file upload with
-either the ``bdist_egg`` filetype or the ``.egg`` suffix, and support for
-uploading Egg distributions will cease to exist on PyPI.
+During the deprecation period users will continue to be allowed to upload egg
+distributions to new and existing projects. Uploading an egg distribution
+during this period will result in an email to all maintainers of the project,
+which will inform them of the upcoming restrictions on egg distribution
+uploads.
 
-As with :pep:`527`, this process is intended to ensure minimal disruption,
-and will **NOT** affect any existing files on PyPI.
+After the deprecation period, support for uploading egg distributions will
+cease to exist on PyPI.
+
+Prior Art
+---------
+
+The removal process above is based on :pep:`527`'s removal process. However,
+only the process above is authoritative for this PEP.
+
+Backwards Compatibility
+=======================
+
+Limited Impact
+--------------
+
+As noted in the rationale section, this PEP is expected to have no impact
+on the overwhelming majority of PyPI users.
```

Comment:
```suggestion
on the overwhelming majority of PyPI users and projects, and there has been
`substantial community coordination <https://github.com/pypi/warehouse/issues/10653>`_
over the past 1 1/2 years to minimize the impact on the few last use cases.
```

You might want to add an explicit mention here of the substantial coordination work that's gone on with the community over the past 1.5 years to minimize the remaining impact.